src/shared/components/QuickLaneModal/QuickLaneModalSharingTab.tsx

```diff
@@ -34,9 +39,20 @@ const QuickLaneModalSharingTab: FunctionComponent<QuickLaneModalProps & UserProp
 }) => {
 	const { tText, tHtml } = useTranslation();
 
+	const fragmentDuration = useMemo(
+		() => toSeconds((content as ItemSchema).duration) || 0,
+		[content]
+	);
+
 	const [quickLane, setQuickLane] = useState<QuickLaneUrlObject>(defaultQuickLaneState);
 	const [exists, setExists] = useState<boolean>(false);
 	const [synced, setSynced] = useState<boolean>(false);
+	const [fragmentStartTime, setFragmentStartTime] = useState<number>(
+		quickLane.start_oc || defaultQuickLaneState.start_oc || 0
+	);
+	const [fragmentEndTime, setFragmentEndTime] = useState<number>(
+		quickLane.end_oc || fragmentDuration || defaultQuickLaneState.end_oc || 0
```

Comment:
be careful that the end time can never be before the start time
or the range component in the time crop controls will throw an exception that causes the page to crash
But maybe this is already checked inside the time crop controls component?

Reply:
Will check